### PR TITLE
Send a discord msg on payment action required

### PR DESCRIPTION
**Pull request by tspenov**
## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have tried to find clearer solution before commenting hard-to-understand parts of code
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: lib/sanbase/billing/subscription/event_handling/discord_notification.ex [hunk 1]
```diff
@@ -43,6 +43,30 @@ defmodule Sanbase.Billing.DiscordNotification do
...
+    customer = stripe_event["data"]["object"]["customer"]
+    customer = "https://dashboard.stripe.com/customers/#{customer}"
+
+    message = """
+    ⛔ A payment for #{formatted_amount} requires action.
+    Customer: #{customer}
+    Event: https://dashboard.stripe.com/events/#{stripe_event_id}
+    """
+
+    do_send_to_discord(message, "Stripe Payment", webhook: "bug_report")
```
Comment (IvanIvanoff) [inline lib/sanbase/billing/subscription/event_handling/discord_notification.ex:67]: is this always happening because of bugs?

**Review by IvanIvanoff — APPROVED** [review]

File: lib/sanbase/billing/subscription/event_handling/discord_notification.ex (hunk 1 above)
Comment (tspenov) [reply lib/sanbase/billing/subscription/event_handling/discord_notification.ex:67]: It is not a bug but it needs an action from our side so there will be more visible. Other option is to tag someone.

Comment (IvanIvanoff) [reply lib/sanbase/billing/subscription/event_handling/discord_notification.ex:67]: I was confused by the `bug_report` webhook name. Maybe `payment_action_required` would be better?

Comment (tspenov) [reply lib/sanbase/billing/subscription/event_handling/discord_notification.ex]: changed

**Pull request merged**